lib/fathead/mdn_css/parse.pl

```diff
@@ -441,10 +441,15 @@ sub parse_initial_value {
                 #get text not in ul
                 $initial_value .= $td->at('ul')->remove->all_text;
 
-                #add new line after each li text so that it appears correctly
-                for my $li ( $ul->find('li')->each ) {
-                    $initial_value .= $li->all_text . '\\n ';
+                my $li_collection = $ul->find('li');
+                my $last_li       = $li_collection->last;
+                $li_collection->last->remove;
+
+                #Separate multiple values by commas
+                for my $li ( $li_collection->each ) {
```

Comment:
@hchienjo we may be able to copy the exact HTML used in the docs here. If we take the content inside the `<td>` and put it inside our `<em>` I think this will work. That way we can display a `<ul>` instead of a textual list.

Wanna give that a try? I will have to deploy to Beta to confirm it works as expected.

We may also need to strip the `<a>` and `<code>` tags out of the `<li>`